chess_library.py

```diff
@@ -1,41 +1,18 @@
-"""
-What's up Youtube, today I'm going to be bringing you a playthrough of 
-a third party chess library. Hopefully I can get some good clips for
-you guys today.
+from code import chess_move
```

Comment:
Look at assignment again. It should be `from chess_move import ChessMove`. Change next line as well